### DISCUSSION TITLE
Select an option

### DISCUSSION THREAD

**Discussion by ghost** [wireui/wireui#158] [General]
Hi,

Is it possible to select an option at the load of the page, with the HTML attribute selected for example or in another way ? 

I searched a little bit but couldn't find a solution to do it

Thank's

**Comment by ph7jack** [comment]
@maximilienl Can you explain it?

**Reply by ghost** [reply]
I achieved what I wanted to do. I am new to Livewire and was unable to assign a value to a property on the mount. But I understood that it was necessary to remove the .defer from the wire: model for it to work
